packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -231,6 +232,7 @@ type LabelStyle {
   borderStyle: LineStyle!
   background: String!
   maxWidth: String
+  visibility: String!
```

Comment:
Why is the visibility just a random string? What would `tralala` mean in this regard?